src/pages/board/[slug].tsx

```diff
@@ -57,14 +58,36 @@ export const getServerSideProps: GetServerSideProps<BoardGetServerSideProps> = a
   const config = await getFrontendConfig(routeParams.data.slug);
   const translations = await getServerSideTranslations(boardNamespaces, locale, req, res);
 
-  return {
-    props: {
-      config,
-      primaryColor: config.settings.customization.colors.primary,
-      secondaryColor: config.settings.customization.colors.secondary,
-      primaryShade: config.settings.customization.colors.shade,
-      dockerEnabled: !!env.DOCKER_HOST && !!env.DOCKER_PORT,
-      ...translations,
-    },
-  };
+  const getSuccessResponse = () => {
+    return {
+      props: {
+        config,
+        primaryColor: config.settings.customization.colors.primary,
+        secondaryColor: config.settings.customization.colors.secondary,
+        primaryShade: config.settings.customization.colors.shade,
+        dockerEnabled: !!env.DOCKER_HOST && !!env.DOCKER_PORT,
+        ...translations,
+      },
+    };
+  }
+
+
+  if (!config.settings.access.allowGuests) {
```

Comment:
You could revert this if and place the success response inside, but only if you want